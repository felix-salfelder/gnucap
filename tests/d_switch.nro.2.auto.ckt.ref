* switch as negative resistance oscillator 
#Time       V(1)       r(SW1)     iter(0)    input(SW1) timef(SW1) control(0)
 0.         0.         10.Meg     0.         0.         Inf        1.        
 2.u        0.19996    10.Meg     3.         0.         49.76u     9.        
 4.u        0.3999     10.Meg     3.         0.19996    49.765u    6.        
 6.u        0.5998     10.Meg     3.         0.3999     49.774u    6.        
 8.u        0.79966    10.Meg     3.         0.5998     49.783u    6.        
 10.u       0.99948    10.Meg     3.         0.79966    49.791u    6.        
 49.791u    4.9667     10.Meg     3.         0.99948    49.874u    7.        
 49.874u    4.975      10.Meg     3.         4.9667     49.874u    7.        
 49.874u    4.975      10.Meg     2.         4.975      49.874u    7.        
 49.874u    4.975      10.Meg     2.         4.975      49.874u    7.        
 49.874u    4.9651     1.         2.         4.975      49.875u    7.        
 49.874u    4.8959     1.         2.         4.9651     49.875u    6.        
 49.875u    3.5786     1.         2.         4.8959     49.875u    6.        
 49.875u    1.4641     1.         2.         3.5786     49.876u    7.        
 49.876u    0.92656    1.         2.         1.4641     49.877u    6.        
 49.876u    0.5864     1.         2.         0.92656    49.877u    6.        
 49.877u    0.41318    1.         2.         0.5864     49.878u    6.        
 49.877u    0.29114    1.         2.         0.41318    49.878u    6.        
 49.877u    0.20516    1.         2.         0.29114    49.878u    6.        
 49.878u    0.14458    1.         2.         0.20516    49.878u    6.        
 49.878u    0.10189    1.         2.         0.14458    49.879u    6.        
 49.878u    0.071819   1.         2.         0.10189    49.879u    6.        
 49.879u    0.05063    1.         2.         0.071819   49.879u    6.        
 49.879u    0.035702   1.         2.         0.05063    49.879u    6.        
 49.879u    0.027833   1.         2.         0.035702   49.879u    7.        
 49.879u    0.025458   1.         2.         0.027833   49.88u     7.        
 49.88u     0.025024   1.         2.         0.025458   49.88u     7.        
 49.88u     0.024974   1.         2.         0.025024   49.88u     7.        
 49.88u     0.024975   10.Meg     2.         0.024974   99.381u    7.        
 49.883u    0.025324   10.Meg     2.         0.024975   99.381u    6.        
 63.53u     1.3891     10.Meg     3.         0.025324   99.415u    6.        
 99.415u    4.9661     10.Meg     3.         1.3891     99.504u    7.        
 99.504u    4.975      10.Meg     2.         4.9661     99.504u    7.        
 99.504u    4.975      10.Meg     2.         4.975      99.504u    7.        
 99.504u    4.975      10.Meg     2.         4.975      99.504u    7.        
 99.504u    4.9651     1.         2.         4.975      99.505u    7.        
 99.504u    4.8959     1.         2.         4.9651     99.505u    6.        
 99.504u    3.5786     1.         2.         4.8959     99.505u    6.        
 99.505u    1.4641     1.         2.         3.5786     99.506u    7.        
 99.506u    0.92656    1.         2.         1.4641     99.506u    6.        
 99.506u    0.5864     1.         2.         0.92656    99.507u    6.        
 99.506u    0.41318    1.         2.         0.5864     99.507u    6.        
 99.507u    0.29114    1.         2.         0.41318    99.507u    6.        
 99.507u    0.20516    1.         2.         0.29114    99.508u    6.        
 99.507u    0.14458    1.         2.         0.20516    99.508u    6.        
 99.508u    0.10189    1.         2.         0.14458    99.508u    6.        
 99.508u    0.071819   1.         2.         0.10189    99.509u    6.        
 99.508u    0.05063    1.         2.         0.071819   99.509u    6.        
 99.509u    0.035702   1.         2.         0.05063    99.509u    6.        
 99.509u    0.027833   1.         2.         0.035702   99.509u    7.        
 99.509u    0.025458   1.         2.         0.027833   99.509u    7.        
 99.509u    0.025024   1.         2.         0.025458   99.509u    7.        
 99.509u    0.024974   1.         2.         0.025024   99.509u    7.        
 99.509u    0.024975   10.Meg     2.         0.024974   149.01u    7.        
 99.513u    0.025324   10.Meg     2.         0.024975   149.01u    6.        
 113.87u    1.4598     10.Meg     3.         0.025324   149.05u    6.        
 149.05u    4.9663     10.Meg     3.         1.4598     149.13u    7.        
 149.13u    4.975      10.Meg     2.         4.9663     149.13u    7.        
 149.13u    4.975      10.Meg     2.         4.975      149.13u    7.        
 149.13u    4.975      10.Meg     2.         4.975      149.13u    7.        
 149.13u    4.9651     1.         2.         4.975      149.13u    7.        
 149.13u    4.8959     1.         2.         4.9651     149.13u    6.        
 149.13u    3.5786     1.         2.         4.8959     149.13u    6.        
 149.13u    1.4641     1.         2.         3.5786     149.14u    7.        
 149.14u    0.92656    1.         2.         1.4641     149.14u    6.        
 149.14u    0.5864     1.         2.         0.92656    149.14u    6.        
 149.14u    0.41318    1.         2.         0.5864     149.14u    6.        
 149.14u    0.29114    1.         2.         0.41318    149.14u    6.        
 149.14u    0.20516    1.         2.         0.29114    149.14u    6.        
 149.14u    0.14458    1.         2.         0.20516    149.14u    6.        
 149.14u    0.10189    1.         2.         0.14458    149.14u    6.        
 149.14u    0.071819   1.         2.         0.10189    149.14u    6.        
 149.14u    0.05063    1.         2.         0.071819   149.14u    6.        
 149.14u    0.035702   1.         2.         0.05063    149.14u    6.        
 149.14u    0.027833   1.         2.         0.035702   149.14u    7.        
 149.14u    0.025458   1.         2.         0.027833   149.14u    7.        
 149.14u    0.025024   1.         2.         0.025458   149.14u    7.        
 149.14u    0.024974   1.         2.         0.025024   149.14u    7.        
 149.14u    0.024975   10.Meg     2.         0.024974   198.64u    7.        
 149.14u    0.025324   10.Meg     2.         0.024975   198.64u    6.        
 161.86u    1.2959     10.Meg     3.         0.025324   198.67u    6.        
 198.67u    4.9659     10.Meg     3.         1.2959     198.76u    7.        
 198.76u    4.975      10.Meg     2.         4.9659     198.76u    7.        
 198.76u    4.975      10.Meg     2.         4.975      198.76u    7.        
 198.76u    4.975      10.Meg     2.         4.975      198.76u    7.        
 198.76u    4.9651     1.         2.         4.975      198.76u    7.        
 198.76u    4.8959     1.         2.         4.9651     198.76u    6.        
 198.76u    3.5787     1.         2.         4.8959     198.76u    6.        
 198.76u    1.4641     1.         2.         3.5787     198.76u    7.        
 198.76u    0.92659    1.         2.         1.4641     198.77u    6.        
 198.77u    0.58643    1.         2.         0.92659    198.77u    6.        
 198.77u    0.41323    1.         2.         0.58643    198.77u    6.        
 198.77u    0.29119    1.         2.         0.41323    198.77u    6.        
 198.77u    0.2052     1.         2.         0.29119    198.77u    6.        
 198.77u    0.14462    1.         2.         0.2052     198.77u    6.        
 198.77u    0.10193    1.         2.         0.14462    198.77u    6.        
 198.77u    0.071848   1.         2.         0.10193    198.77u    6.        
 198.77u    0.050653   1.         2.         0.071848   198.77u    6.        
 198.77u    0.03572    1.         2.         0.050653   198.77u    6.        
 198.77u    0.027838   1.         2.         0.03572    198.77u    7.        
 198.77u    0.02546    1.         2.         0.027838   198.77u    7.        
 198.77u    0.025024   1.         2.         0.02546    198.77u    7.        
 198.77u    0.024975   1.         2.         0.025024   198.77u    7.        
 198.77u    0.024975   10.Meg     2.         0.024975   248.27u    7.        
 198.77u    0.025324   10.Meg     2.         0.024975   248.27u    6.        
 200.u      0.14814    10.Meg     3.         0.025324   248.27u    1.        
Gnucap   System status
iterations: op=0, dc=0, tran=224, fourier=0, total=330
transient timesteps: accepted=106, rejected=0, total=106
nodes: user=1, subckt=0, model=0, total=1
dctran density=100.0%, ac density=100.0%
