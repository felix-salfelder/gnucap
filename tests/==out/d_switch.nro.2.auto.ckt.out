* switch as negative resistance oscillator 
#Time       V(1)       r(SW1)     iter(0)    input(SW1) timef(SW1) control(0)
 0.         0.         10.Meg     0.         0.         Inf        1.        
 2.u        0.19996    10.Meg     3.         0.         49.76u     9.        
 4.u        0.3999     10.Meg     3.         0.19996    49.765u    6.        
 6.u        0.5998     10.Meg     3.         0.3999     49.774u    6.        
 8.u        0.79966    10.Meg     3.         0.5998     49.783u    6.        
 10.u       0.99948    10.Meg     3.         0.79966    49.791u    6.        
 49.791u    4.9667     10.Meg     3.         0.99948    49.874u    7.        
 49.874u    4.975      10.Meg     3.         4.9667     49.874u    7.        
 49.874u    4.975      10.Meg     2.         4.975      49.874u    7.        
 49.874u    4.975      10.Meg     2.         4.975      49.874u    7.        
 49.874u    4.97       1.         2.         4.975      49.876u    7.        
 49.874u    4.839      1.         2.         4.97       49.875u    6.        
 49.874u    4.7629     1.         2.         4.839      49.875u    6.        
 49.874u    4.6099     1.         2.         4.7629     49.875u    6.        
 49.875u    3.0311     1.         2.         4.6099     49.876u    6.        
 49.875u    1.993      1.         2.         3.0311     49.876u    6.        
 49.876u    1.3104     1.         2.         1.993      49.876u    6.        
 49.876u    0.86165    1.         2.         1.3104     49.877u    6.        
 49.876u    0.56658    1.         2.         0.86165    49.877u    6.        
 49.877u    0.37256    1.         2.         0.56658    49.878u    6.        
 49.877u    0.245      1.         2.         0.37256    49.878u    6.        
 49.878u    0.16112    1.         2.         0.245      49.878u    6.        
 49.878u    0.10597    1.         2.         0.16112    49.879u    6.        
 49.879u    0.069712   1.         2.         0.10597    49.879u    6.        
 49.879u    0.04587    1.         2.         0.069712   49.879u    6.        
 49.879u    0.031849   1.         2.         0.04587    49.879u    7.        
 49.879u    0.026694   1.         2.         0.031849   49.88u     7.        
 49.88u     0.025193   1.         2.         0.026694   49.88u     7.        
 49.88u     0.025006   1.         2.         0.025193   49.88u     7.        
 49.88u     0.024957   1.         2.         0.025006   49.88u     7.        
 49.88u     0.024932   10.Meg     2.         0.024957   Inf        7.        
 49.88u     0.024933   10.Meg     2.         0.024932   99.381u    6.        
 49.88u     0.024934   10.Meg     2.         0.024933   99.381u    6.        
 49.88u     0.024936   10.Meg     2.         0.024934   99.381u    6.        
 49.999u    0.036878   10.Meg     3.         0.024936   99.382u    6.        
 50.745u    0.1115     10.Meg     3.         0.036878   99.384u    6.        
 63.183u    1.3544     10.Meg     3.         0.1115     99.416u    6.        
 99.416u    4.9662     10.Meg     3.         1.3544     99.504u    7.        
 99.504u    4.975      10.Meg     2.         4.9662     99.504u    7.        
 99.504u    4.975      10.Meg     2.         4.975      99.504u    7.        
 99.504u    4.975      10.Meg     2.         4.975      99.504u    7.        
 99.504u    4.97       1.         2.         4.975      99.506u    7.        
 99.504u    4.8404     1.         2.         4.97       99.505u    6.        
 99.504u    4.7645     1.         2.         4.8404     99.505u    6.        
 99.504u    4.6123     1.         2.         4.7645     99.505u    6.        
 99.505u    3.0321     1.         2.         4.6123     99.506u    6.        
 99.505u    1.9933     1.         2.         3.0321     99.506u    6.        
 99.506u    1.3104     1.         2.         1.9933     99.506u    6.        
 99.506u    0.86148    1.         2.         1.3104     99.507u    6.        
 99.506u    0.56636    1.         2.         0.86148    99.507u    6.        
 99.507u    0.37235    1.         2.         0.56636    99.508u    6.        
 99.507u    0.24482    1.         2.         0.37235    99.508u    6.        
 99.508u    0.16097    1.         2.         0.24482    99.508u    6.        
 99.508u    0.10586    1.         2.         0.16097    99.509u    6.        
 99.509u    0.069623   1.         2.         0.10586    99.509u    6.        
 99.509u    0.045803   1.         2.         0.069623   99.509u    6.        
 99.509u    0.031824   1.         2.         0.045803   99.51u     7.        
 99.51u     0.026685   1.         2.         0.031824   99.51u     7.        
 99.51u     0.025192   1.         2.         0.026685   99.51u     7.        
 99.51u     0.025006   1.         2.         0.025192   99.51u     7.        
 99.51u     0.024956   1.         2.         0.025006   99.51u     7.        
 99.51u     0.024932   10.Meg     2.         0.024956   Inf        7.        
 99.51u     0.024933   10.Meg     2.         0.024932   149.01u    6.        
 99.51u     0.024934   10.Meg     2.         0.024933   149.01u    6.        
 99.51u     0.024936   10.Meg     2.         0.024934   149.01u    6.        
 99.573u    0.03128    10.Meg     2.         0.024936   149.01u    6.        
 100.18u    0.092141   10.Meg     3.         0.03128    149.01u    6.        
 108.5u     0.92354    10.Meg     3.         0.092141   149.04u    6.        
 149.04u    4.9651     10.Meg     3.         0.92354    149.13u    7.        
 149.13u    4.975      10.Meg     2.         4.9651     149.13u    7.        
 149.13u    4.975      10.Meg     2.         4.975      149.13u    7.        
 149.13u    4.975      10.Meg     2.         4.975      149.13u    7.        
 149.13u    4.97       1.         2.         4.975      149.14u    7.        
 149.13u    4.8306     1.         2.         4.97       149.14u    6.        
 149.13u    4.7534     1.         2.         4.8306     149.14u    6.        
 149.13u    4.5956     1.         2.         4.7534     149.14u    6.        
 149.13u    3.0247     1.         2.         4.5956     149.14u    6.        
 149.14u    1.9908     1.         2.         3.0247     149.14u    6.        
 149.14u    1.3103     1.         2.         1.9908     149.14u    6.        
 149.14u    0.86247    1.         2.         1.3103     149.14u    6.        
 149.14u    0.56769    1.         2.         0.86247    149.14u    6.        
 149.14u    0.37367    1.         2.         0.56769    149.14u    6.        
 149.14u    0.24598    1.         2.         0.37367    149.14u    6.        
 149.14u    0.16193    1.         2.         0.24598    149.14u    6.        
 149.14u    0.10661    1.         2.         0.16193    149.14u    6.        
 149.14u    0.070204   1.         2.         0.10661    149.14u    6.        
 149.14u    0.046241   1.         2.         0.070204   149.14u    6.        
 149.14u    0.031984   1.         2.         0.046241   149.14u    7.        
 149.14u    0.026745   1.         2.         0.031984   149.14u    7.        
 149.14u    0.025202   1.         2.         0.026745   149.14u    7.        
 149.14u    0.025007   1.         2.         0.025202   149.14u    7.        
 149.14u    0.024957   1.         2.         0.025007   149.14u    7.        
 149.14u    0.024932   10.Meg     2.         0.024957   Inf        7.        
 149.14u    0.024933   10.Meg     2.         0.024932   198.64u    6.        
 149.14u    0.024934   10.Meg     2.         0.024933   198.64u    6.        
 149.14u    0.024937   10.Meg     2.         0.024934   198.64u    6.        
 149.31u    0.042235   10.Meg     3.         0.024937   198.64u    6.        
 150.05u    0.11569    10.Meg     3.         0.042235   198.64u    6.        
 160.04u    1.1141     10.Meg     3.         0.11569    198.67u    6.        
 198.67u    4.9656     10.Meg     3.         1.1141     198.76u    7.        
 198.76u    4.975      10.Meg     2.         4.9656     198.76u    7.        
 198.76u    4.975      10.Meg     2.         4.975      198.76u    7.        
 198.76u    4.975      10.Meg     2.         4.975      198.76u    7.        
 198.76u    4.97       1.         2.         4.975      198.77u    7.        
 198.76u    4.835      1.         2.         4.97       198.77u    6.        
 198.76u    4.7584     1.         2.         4.835      198.77u    6.        
 198.76u    4.6031     1.         2.         4.7584     198.77u    6.        
 198.76u    3.0285     1.         2.         4.6031     198.77u    6.        
 198.77u    1.9925     1.         2.         3.0285     198.77u    6.        
 198.77u    1.311      1.         2.         1.9925     198.77u    6.        
 198.77u    0.86254    1.         2.         1.311      198.77u    6.        
 198.77u    0.56751    1.         2.         0.86254    198.77u    6.        
 198.77u    0.37341    1.         2.         0.56751    198.77u    6.        
 198.77u    0.24571    1.         2.         0.37341    198.77u    6.        
 198.77u    0.16169    1.         2.         0.24571    198.77u    6.        
 198.77u    0.10641    1.         2.         0.16169    198.77u    6.        
 198.77u    0.070045   1.         2.         0.10641    198.77u    6.        
 198.77u    0.046118   1.         2.         0.070045   198.77u    6.        
 198.77u    0.031939   1.         2.         0.046118   198.77u    7.        
 198.77u    0.026728   1.         2.         0.031939   198.77u    7.        
 198.77u    0.025199   1.         2.         0.026728   198.77u    7.        
 198.77u    0.025007   1.         2.         0.025199   198.77u    7.        
 198.77u    0.024957   1.         2.         0.025007   198.77u    7.        
 198.77u    0.024932   10.Meg     2.         0.024957   Inf        7.        
 198.77u    0.024933   10.Meg     2.         0.024932   248.27u    6.        
 198.77u    0.024934   10.Meg     2.         0.024933   248.27u    6.        
 198.77u    0.024936   10.Meg     2.         0.024934   248.27u    6.        
 198.83u    0.030795   10.Meg     2.         0.024936   248.27u    6.        
 199.41u    0.089378   10.Meg     3.         0.030795   248.27u    6.        
 200.u      0.14796    10.Meg     3.         0.089378   248.28u    1.        
Gnucap   System status
iterations: op=0, dc=0, tran=278, fourier=0, total=408
transient timesteps: accepted=130, rejected=0, total=130
nodes: user=1, subckt=0, model=0, total=1
dctran density=100.0%, ac density=100.0%
